desktop/app-handlers/crash-reporting/index.js

```diff
@@ -11,10 +11,10 @@ const debug = require( 'debug' )( 'desktop:crash-reporting' );
 /**
  * Internal dependencies
  */
-const Config = require( 'lib/config' );
+const Config = require( '../../lib/config' );
```

Comment:
Agreed :)
I don't see any problem with a relative path for desktop's internals!
An alternative that I've been playing with is just aliasing `desktop` so that this becomes:
```js
const Config = require( 'desktop/lib/config' );
```
Though I don't think this actually adds _much_ value over a relative path, beyond just allowing further nested files use the same reference and avoid the `'../../../../xx` type paths.